prometheus-exporters/openstack-exporter/templates/openstack-exporter-openstack-config.yaml

```diff
@@ -17,6 +17,12 @@ data:
       cinderbackend:
         collector: openstack_exporter.collectors.cinderbackend.CinderBackendCollector
         enabled: True
+        {{- if .Values.collectors.cinderbackend.expected_sharding_backends }}
+        expected_sharding_backends: {{- .Values.collectors.cinderbackend.expected_sharding_backends }}
+        {{- end }}
+        {{- if .Values.collectors.cinderbackend.allow_unexpected_backends }}
```

Comment:
This prohibits us from setting an explicit `False` value. It would only allow us to get the default from `openstack-exporter`, which is currently defaulting to `False`.

Renaming the variable to `set_allow_unexpected_backends` would transport that information better.

Reply:
updated